sample/HttpTrigger-CSharp-Poco/run.csx

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Azure.WebJobs.Host;
+
+public class RequestData
+{
+    public string Id { get; set; }
+    public string Value { get; set; }
+}
+
+public static HttpResponseMessage Run(RequestData data, HttpRequestMessage req, out string outBlob, TraceWriter log)
```

Comment:
Since a POCO binding is required to get binding data support, I'm also adding the ability for functions to bind to the raw request as well, if they need additional request info.